src/solarized8.vim

```diff
@@ -466,6 +466,17 @@ for s:solarized_background in ["dark", "light"]
     call s:put("hi! ColorColumn"    .s:fmt_none   .s:fg_none   .s:bg_base02)
     call s:put("hi! Cursor"         .s:fmt_none   .s:fg_base3  .s:bg_blue)
     call s:put("hi! link lCursor Cursor")
+
+    call s:put("if exists(':terminal')")
```

Comment:
History shows that it's likely that the similar features in Vim and NeoVim, which are developed independently, are somewhat different (see jobs, for instance). So, I'd rather keep this code under an explicit check for NeoVim. When (and if) Vim implements an embedded terminal, we will might consider generalizing the test.